vendor/k8s.io/kubernetes/pkg/scheduler/algorithmprovider/defaults/compatibility_test.go

```diff
@@ -835,6 +835,7 @@ func TestCompatibility_v1_Scheduler(t *testing.T) {
 			{"name": "MaxEBSVolumeCount"},
 			{"name": "MaxGCEPDVolumeCount"},
 			{"name": "MaxAzureDiskVolumeCount"},
+                        {"name": "MaxCinderVolumeCount"},
```

Comment:
Looks like the indentation is off a bit...

Reply:
is this github thing? I don't see indentation being off....

![screenshot from 2019-02-08 11-19-55](https://user-images.githubusercontent.com/278/52490894-d24d9f00-2b93-11e9-94d1-4f335fe7640b.png)

Reply:
Alright you nitpickers - I did not use tabs for indenting this one and use whitespaces and that is why indentation looked bit off. Should be fixed. :-)